crates/rooch/src/commands/move_cli/commands/new.rs

```diff
@@ -21,13 +25,53 @@ const ROOCH_FRAMEWORK_PKG_PATH: &str = "{ git = \"https://github.com/rooch-netwo
 
 #[derive(Parser)]
 pub struct New {
+    /// Existing account address from Rooch
+    #[clap(long = "address", short = 'a')]
+    account_address: Option<AccountAddress>,
+
     #[clap(flatten)]
     pub new: new::New,
 }
 
 impl New {
-    pub fn execute(self, path: Option<PathBuf>) -> anyhow::Result<()> {
+    async fn get_active_account_address_from_config(&self) -> Result<String, RoochError> {
```

Comment:
You should use `WalletContextOptions` to build the WalletContext, Ref

https://github.com/rooch-network/rooch/blob/17e30cbfe5bc4de3a16ba3440b2d874b1a899588/crates/rooch/src/commands/move_cli/commands/build.rs#L18-L35

Reply:
Added WalletContextOptions.